include/z64.h

```diff
@@ -861,13 +800,6 @@ typedef struct GameState {
     /* 0xA3 */ u8 unk_A3;
 } GameState; // size = 0xA4
 
-typedef struct {
-    /* 0x00 */ u32 resetting;
-    /* 0x04 */ u32 resetCount;
-    /* 0x08 */ OSTime duration;
-    /* 0x10 */ OSTime resetTime;
-} PreNmiBuff; // size = 0x18 (actually osAppNmiBuffer is 0x40 bytes large but the rest is unused)
-
```

Comment:
So this is just a duplicate of `NmiBuff`? I think the whole phrase `PreNmi` means reset, is just `Nmi` okay? Should `PreNmiBuff` be the struct used and `NmiBuff` be the one that's deleted?

Reply:
if it's the same as in oot, `PreNmiBuff` would make more sense
though the names are vaguely jank since libultra uses "nmibuffer" (no pre-) :shrug: 
http://n64devkit.square7.ch/n64man/os/osAppNMIBuffer.htm

Reply:
This sounds like @Thar0 territory